content/posts/2023/reana-client-prune-command.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Free up disk space in REANA workspaces with the new prune command"
+date: 2023-04-24T07:00:00+01:00
+---
+
+The new prune command in the reana-client software allows users to easily
+free up disk space in their REANA workspaces by deleting temporary files
+that are neither inputs nor outputs. This can help manage cluttered
+workspaces and prevent running out of disk space. 
+
+<!--more-->
+
+
+## Enable Kerberos for a single workflow step
+
+The management of disk space in REANA can be a challenge, especially when the workspace is
+cluttered by numerous temporary files that are no longer needed.
+If you have been looking for a way to easily free up disk space in your workspaces,
+look no further than the new `prune` command in `reana-client`!
+
+With the `prune` command, you can easily delete all the files from your workspace that are
+neither input nor output files. This can free up a significant amount of disk space and make
+your workspace much more manageable. Plus, if you need to free up even more space, you can use the
+`--include-inputs` and `--include-outputs` flags to delete those kinds of files too.
+
+So how do you use the `prune` command? It's easy! Just run `reana-client prune -w <workflow_name>` to
+delete all the files from the workspace of the specified workflow that are neither input nor output files.
+If you want to include input or output files in the deletion process,
+use the `--include-inputs` or `--include-outputs` flags, respectively.
+
+However, it's important to note that running the prune command will permanently delete the files
+from your workspace. This means that you should make sure you have backed up any files you want to keep
+before running the command. Additionally, be careful when using the `--include-inputs` or
+`--include-outputs` flags, as deleting input or output files could impact the results of your workflow.
```

Comment:
```suggestion
before running the command. Additionally, be careful when using the `--include-inputs` or
`--include-outputs` flags, as deleting inputs will make it impossible to run your workflow again and by deleting output files you will lose the results of your workflow run.
```

What do you think of describing what happens more precisely, instead of a more generic _could impact the results of your wofklow_?